### PR TITLE
Added german translation + prettier

### PR DESCRIPTION
**Pull request by MicahBCode**
I've added the german translation for all current pages.
Additionally I've ran prettier over the whole code.

### CODE REVIEW

**Review by katherinepeterson — CHANGES_REQUESTED** [review]
Can you resolve conflicts?

**Comment by MicahBCode** [comment]
@katherinepeterson Resolved. 👍

File: public/locales/de/editor.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "editor-desktop-optimized": "Diese Seite ist für Desktop optimiert",
+  "editor-visit-desktop": "Bitte besuche readme.so auf einem Desktop um deine Readme zu erstellen!",
+  "nav-download": "Herunterladen",
+  "download-readme-generated": "Readme generiert!",
```
Comment (mheob) [inline public/locales/de/editor.json:5]:
```suggestion
  "download-readme-generated": "Readme erstellt!",
```

I think `Readme erstellt!` sounds more natural than `Readme generiert!`.

File: public/locales/de/editor.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "download-reach-out": "Danke, dass du readme.so verwendest! Fühl dich frei, mich über",
+  "download-feedback": "für Rückmeldungen zu kontaktieren.",
+  "download-coffee": "Wenn du dieses Produkt hilfreich findest, unterstütze mich doch gerne mit einer Tasse Kaffe!",
+  "editor-column-editor": "Editor",
+  "editor-select": "Wähle einen Abschnitt aus der linken Seitennavigation um den Inhalt zu bearbeiten",
+  "preview-column-preview": "Vorschau",
+  "preview-column-raw": "Unbearbeitet",
+  "section-column-section": "Abschnitte",
+  "section-column-click-edit": "Klicke auf einen Abschnitt unterhalb, um den Inhalt zu bearbeiten",
+  "section-column-click-add": "Klicke auf einen Abschnitt unterhalb, um ihn zu deiner Readme hinzuzufügen"
```
Comment (mheob) [inline public/locales/de/editor.json:15]:
```suggestion
  "section-column-click-edit": "Klicke auf einen der unteren Abschnitte, um den Inhalt zu bearbeiten",
  "section-column-click-add": "Klicke auf einen der unteren Abschnitte, um ihn zu deiner Readme hinzuzufügen"
```

Here I also think it sounds more natural.

**Review by mheob — COMMENTED** [review]
As a native German speaker, I added a suggestion in two places.

All other translations look good to me.

File: public/locales/de/editor.json (hunk 1 above)
Comment (MicahBCode) [reply public/locales/de/editor.json:5]: Sounds good for me as Swiss German native too.

**Review by mheob — APPROVED** [review]

File: public/locales/de/common.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "title": "Der einfachste Weg zu einer",
+  "redme": "README",
```
Comment (HannesOlszewski) [inline public/locales/de/common.json:3]:
```suggestion
  "readme": "README",
```

**Comment by MicahBCode** [comment]
Everything is set. Ready to be merged 👍

**Pull request merged**